builders/build/historical-updates.md

```diff
@@ -0,0 +1,517 @@
+
+This migration was executed at the following runtimes and blocks:
+
+|    Network     | Executed Runtime | Block Applied |
+|:--------------:|:----------------:|:-------------:|
+|    Moonbeam    |      RT1502      |    1107285    |
+|   Moonriver    |      RT1502      |    1814458    |
+| Moonbase Alpha |      RT1502      |    2112058    |
+
+For more information, you can review the [relative PR on GitHub](https://github.com/PureStake/moonbeam/pull/1408){target=_blank}.
+
+***
+
+#### Replace Staking Reserves with Locks {: #replace-staking-reserves }
+
+A migration was applied that changed users' staking reserved balances to locked balances. The locked balance is the same type as democracy-locked funds, making it possible for users to use their staked funds to participate in democracy.
+
+This migration was executed at the following runtimes and blocks:
+
+|    Network     | Executed Runtime | Block Applied |
+|:--------------:|:----------------:|:-------------:|
+|    Moonbeam    |      RT1701      |    1581457    |
+|   Moonriver    |      RT1701      |    2281723    |
+| Moonbase Alpha |      RT1700      |    2529736    |
+
+For more information, you can review the [relative PR on GitHub](https://github.com/PureStake/moonbeam/pull/1604){target=_blank}.
+
+***
+
+#### Auto-Compounding Support {: #auto-compounding-support }
+
+To support auto-compounding, two migrations were applied to the `AtStake` storage item in the parachain staking pallet:
+
+- `RemovePaidRoundsFromAtStake` -  to remove any stale `AtStake` entries relating to already paid-out rounds that had candidates that didn't produce any blocks. This migration is a prerequisite for the `MigrateAtStakeAutoCompound` migration
+- `MigrateAtStakeAutoCompound` -  migrates the snapshots for unpaid rounds for `AtStake` entries
+
+These migrations were executed at the following runtimes and blocks:
+
+|    Network     | Executed Runtime | Block Applied |
+|:--------------:|:----------------:|:-------------:|
+|    Moonbeam    |      RT1901      |    2317683    |
+|   Moonriver    |      RT1901      |    2911863    |
+| Moonbase Alpha |      RT1900      |    3069635    |
+
+For more information, you can review the [relative PR on GitHub](https://github.com/PureStake/moonbeam/pull/1878){target=_blank}.
+
+***
+
+### XCM-Related Pallets {: #xcm-related-pallets }
+
+#### Update Transact Info Storage Item {: #update-transaction-info }
+
+There was a migration applied to the `TransactInfo` storage item of the XCM-transactor pallet that changed the following items:
+
+- `max_weight` is added to prevent transactors from stalling the queue in the destination chain
+- Removes `fee_per_byte`, `metadata_size`, and `base_weight` as these items are not necessary for XCM transactions
+- `fee_per_second` replaces `fee_per_weight` to better reflect cases (like Kusama) in which the `fee_per_eight` unit is lower than one
```

Comment:
`fee_per_eight` should be `fee_per_weight`?

Reply:
Fixed in https://github.com/PureStake/moonbeam-docs/pull/519